_posts/2018-11-12-sr2019-rules-released.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Minor: can we rename this file to more closely match the title? I don't mind if we skip the "& Useful Resources" part from the url (in fact I'd encourage that), but it would be good to change `rules-released` to `rules-published`.